mysql-test/suite/innodb/r/sys_truncate_shutdown_debug.result

```diff
@@ -0,0 +1,39 @@
+call mtr.add_suppression("InnoDB: Cannot shrink the system tablespace");
+call mtr.add_suppression("InnoDB: Plugin initialization aborted");
+call mtr.add_suppression("Plugin 'InnoDB' init function returned error");
+call mtr.add_suppression("Plugin 'InnoDB' registration as a STORAGE ENGINE failed");
+SET GLOBAL INNODB_LIMIT_OPTIMISTIC_INSERT_DEBUG=2;
+SET GLOBAL INNODB_FILE_PER_TABLE= 0;
+Warnings:
+Warning	1287	'@@innodb_file_per_table' is deprecated and will be removed in a future release
+SET UNIQUE_CHECKS=0, FOREIGN_KEY_CHECKS=0;
+CREATE TABLE t1(f1 INT NOT NULL, f2 INT NOT NULL,
+f3 INT NOT NULL, INDEX(f1),
+INDEX(f2), INDEX(f3))ENGINE=InnoDB;
+BEGIN;
+INSERT INTO t1 SELECT seq, seq, seq FROM seq_1_to_16384;
+INSERT INTO t1 SELECT seq, seq, seq FROM seq_1_to_16384;
+COMMIT;
+DROP TABLE t1;
+SELECT NAME, FILE_SIZE FROM INFORMATION_SCHEMA.INNODB_SYS_TABLESPACES WHERE SPACE = 0;
+NAME	FILE_SIZE
+innodb_system	537919488
```

Comment:
This is occasionally reporting a different size. At least 540016640 has been observed.

Could it be that we need a fresh InnoDB bootstrap? Also, the normal bootstrap, which is executing some SQL, could be nondeterministic. Could we let InnoDB create a new data directory, by specifying `--innodb-data-home-dir` in this test? See the test `innodb.log_file` for an example.

Reply:
What was the fix? To add another slow shutdown before the test?